src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -183,6 +200,16 @@ export default {
 			// );
 			// this.matchedLoans = loans;
 		},
+		fetchMFILoans() {
+			// Load mfi recommendations loans data
+			return this.apollo.query({
```

Comment:
The partner ID needs to be added to the spotlight data and passed to this query, since the query still has partner ID hardcoded.

Reply:
This query will be replaced with a flss call passing filter criteria from spotlight data object in this ticket https://kiva.atlassian.net/browse/CORE-1017